### PR TITLE
Announce Week 32

### PR DESCRIPTION
**Pull request by VoyTechnology**
First announce ever, please rip it apart and say mean things.

### CODE REVIEW

File: source/_posts/2017-week-31-announce.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+banner: exams_coming.jpg
+date: 2017-04-27 22:33:55
+author: voy
+tags:
+  - Hoodies
+  - Exams
+---
+Dear Brickies,
+
+Exams in the distance, but we're nearly there! Too cheer you up, the hoodies
```
Comment (zergless) [inline source/_posts/2017-week-31-announce.md]: should it not be to?

File: source/_posts/2017-week-31-announce.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: Announce 31 | New Committee | Good Luck | Hoodies
+banner: exams_coming.jpg
```
Comment (butlerx) [inline source/_posts/2017-week-31-announce.md]: 1:1 images dont work for banners

File: source/_posts/2017-week-31-announce.md (hunk 1 above)
Comment (butlerx) [inline source/_posts/2017-week-31-announce.md]: -the

File: source/_posts/2017-week-31-announce.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	- Helpdesk: Sahil Mehra (pr0n)
+	- Helpdesk: Michal Durinik (mikello)
+
+On behalf of the new committee we would also like to thank the 2016-2017
+committee for their hard work, as well as giving us suggestions and ideas they
+didn't have time to execute. Best of luck!
+
+## Good Luck in the exams
+
+Yes. Exams are coming. On behalf of the new committee, best of luck, you CAN do
```
Comment (butlerx) [inline source/_posts/2017-week-31-announce.md]: you say on behalf of the new committee too much, vary you language

File: source/_posts/2017-week-31-announce.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## Hoodies
+
+The hoodies have (finally) arrived! If you have ordered a hoodie you should
+get an email shortly regarding the collection. If you didn't order, you might
+get a chance next year, so don't worry :)
+
+"How does this announce thing work again?"
+Wojtek Bednarzak (~voy)
+Redbrick Secretary
```
Comment (butlerx) [inline source/_posts/2017-week-31-announce.md]: year?

File: source/_posts/2017-week-31-announce.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+s o c i a l m e d i a
```
Comment (butlerx) [inline source/_posts/2017-week-31-announce.md]: s/ //

File: source/_posts/2017-week-31-announce.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+Twitter: @RedbrickDCU
+Facebook: https://www.facebook.com/dcuredbrick
+Snapchat: RedbrickDCU
+Google Calendar: http://tinyurl.com/redbrickCalendar
```
Comment (butlerx) [inline source/_posts/2017-week-31-announce.md]: why not out event page https://www.redbrick.dcu.ie/events/
or https://events.redbrick.dcu.ie/

File: source/_posts/2017-week-31-announce.md (hunk 5 above)
Comment (VoyTechnology) [inline source/_posts/2017-week-31-announce.md]: I presumed it was on purpose with all previous announces sending it this way

File: source/_posts/2017-week-31-announce.md (hunk 6 above)
Comment (VoyTechnology) [inline source/_posts/2017-week-31-announce.md]: Old footer. Will update

File: source/_posts/2017-week-31-announce.md (hunk 5 above)
Comment (butlerx) [inline source/_posts/2017-week-31-announce.md]: it was the previous sec preference

Comment (VoyTechnology) [inline source/_posts/2017-week-31-announce.md]: I'm going to keep it cause I quite like it :)

**Comment by d-fens** [comment]
I think I've asked this before, but can we stop line wrapping on these posts are it causes weird line breaks on the website and makes it visually very odd to read. 

However if this is used for mail, then we need to investigate that since wrapping will cause issues.

Eg see the line mentioning gist and how there is a break on the website for it:
https://raw.githubusercontent.com/redbrick/static-site/master/source/_posts/2017-week-26-announce.md
https://www.redbrick.dcu.ie/2017/03/13/2017-week-26-announce/

**Comment by butlerx** [comment]
the site can send the email after each new posting. iirc it wraps emails at 80 characters

**Comment by butlerx** [comment]
also is it not week 32

File: source/_posts/2017-week-31-announce.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+## New Committee
+
+As you might already know, with the AGM behind us, the committee has changed.
+We are working relentlessly - reviewing, planning and organising next year to
+make it better than all the previous ones. In case you don't know who is
+responsible for all the mess next year, let me re-introduce you to the new
+committee.
+
+	- Chairperson: Terry Bolt (tbolt)
```
Comment (butlerx) [inline source/_posts/2017-week-31-announce.md]: is this meant to be a code block?

Comment (GoldenBadger) [inline source/_posts/2017-week-31-announce.md]: I'm assuming this is meant to be a bullet list.

**Comment by butlerx** [comment]
https://goo.gl/photos/dMc5WtXdEZHRsoJX9 <- banner image

File: source/_posts/2017-week-32-announce.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+The hoodies have (finally) arrived! If you have ordered a hoodie you should
+get an email shortly regarding the collection. If you didn't order, you might
+get a chance next year, so don't worry :)
+
+"How does this announce thing work again?"
+Wojtek Bednarzak (~voy)
+Redbrick Secretary 2017-2018
+
+s o c i a l m e d i a
```
Comment (GoldenBadger) [inline source/_posts/2017-week-32-announce.md]: When the MD is rendered these all appear on the same line and look ugly.

Comment (VoyTechnology) [inline source/_posts/2017-week-32-announce.md]: Would I remove this section from the page and only send it in email?

Comment (butlerx) [inline source/_posts/2017-week-32-announce.md]: just add  `  ` to the end of the line

**Pull request merged**